plugins/modules/stepfunctions_state_machine.py

```diff
@@ -9,51 +9,53 @@
 
 DOCUMENTATION = '''
 ---
-module: aws_step_functions_state_machine
+module: stepfunctions_state_machine
 version_added: 1.0.0
 short_description: Manage AWS Step Functions state machines
 description:
-  - Create, update and delete state machines in AWS Step Functions.
-  - Calling the module in C(state=present) for an existing AWS Step Functions state machine
-    will attempt to update the state machine definition, IAM Role, or tags with the provided data.
+    - Create, update and delete state machines in AWS Step Functions.
+    - Calling the module in C(state=present) for an existing AWS Step Functions state machine
+      will attempt to update the state machine definition, IAM Role, or tags with the provided data.
+    - Prior to release 5.0.0 this module was called C(community.aws.aws_step_functions_state_machine).
```

Comment:
Has to be C() - Docs generation will try to lookup M(), and since there's no longer a "community.aws.aws_step_functions_state_machine" module it throws a linting error when the docs are linted.